biogrid/apps/biogrid-mvp/src/app/pages/simulate-page/simulate-page.tsx

```diff
@@ -51,64 +53,97 @@ export const SimulatePage = () => {
       return {
         fromItem: edge.v,
         toItem: edge.w,
+        // TODO this is not a correct value
```

Comment:
the comment behind the TODO isn't very actionable. What do we do in order to correct it?

Reply:
Woops, that was a note to myself. It is correct now